Tests/StencilSwiftKitTests/StringFiltersTests.swift

```diff
@@ -133,6 +133,77 @@ class StringFiltersTests: XCTestCase {
     }
   }
 
+  func testCamelToSnakeCase_WithNoArgsDefaultsToTrue() {
```

Comment:
You can remove this case, it's covered by the parseBool tests and the 2 cases below this one.